app/views/doc/rebase.html.haml

```diff
@@ -1,180 +0,0 @@
-          %li= link_to "diff-index"
-          %li= link_to "hash-object"
-          %li= link_to "merge-base"
-          %li= link_to "read-tree"
-          %li= link_to "rev-list"
-          %li= link_to "rev-parse"
-          %li= link_to "show-ref"
-          %li= link_to "symbolic-ref"
-          %li= link_to "update-index"
-          %li= link_to "update-ref"
-          %li= link_to "verify-pack"
-          %li= link_to "write-tree"
-
-
-
-  =link_to "Version 1.7.9.4 ▾", "#", {:class => 'dropdown-trigger', :id => 'reference-versions-trigger', 'data-panel-id' => 'previous-versions-dropdown'}
-  <span class="light">git-rebase last updated in 1.7.7.1</span>
-  %div#previous-versions-dropdown.dropdown-panel.left
-    %header
-      Changes in the <strong>git-rebase</strong> manual
-    %ol.reference-previous-versions
-      %li.no-change
-        <span>1.7.7.1 → 1.7.9.4 no changes</span>
-      %li
-        <a href="#"><span class="version">1.7.7.1</span>
-        <span class="diff">
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/red-dot.png" />
-        <img src="/images/icons/red-dot.png" />
-        <img src="/images/icons/red-dot.png" />
-        <img src="/images/icons/grey-dot.png" />
-        </span>
-        <em class="date">12/22/11</em></a>
-      %li.no-change
-        <span>1.7.6.5 no changes</span>
-      %li
-        <a href="#"><span class="version">1.7.6.1</span>
-        <span class="diff">
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/red-dot.png" />
-        <img src="/images/icons/red-dot.png" />
-        <img src="/images/icons/grey-dot.png" />
-        <img src="/images/icons/grey-dot.png" />
-        <img src="/images/icons/grey-dot.png" />
-        <img src="/images/icons/grey-dot.png" />
-        </span>
-        <em class="date">9/22/11</em></a>
-      %li.no-change
-        <span>1.7.5.1 → 1.7.6.0 no changes</span>
-      %li
-        <a href="#"><span class="version">1.7.5.0</span>
-        <span class="diff">
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/green-dot.png" />
-        <img src="/images/icons/red-dot.png" />
-        <img src="/images/icons/red-dot.png" />
-        <img src="/images/icons/grey-dot.png" />
-        <img src="/images/icons/grey-dot.png" />
-        </span>
-        <em class="date">5/22/11</em></a>
-      %li
-        =link_to "See more previous releases →", "#", {:class => 'more'}
-
-    %footer
-      <span class="light">Check your version of git by running<br></span>
-      <strong>$</strong>
-      <input type="text" class="copyable" value="git --version" />
```

Comment:
this was simply deleted?

Reply:
Yeah, I apparently created this file while working on the layout before we wired everything up.

Reply:
We do use `git-rebase` as our example in some of the tests, so I wondered if it might be necessary there. But I'm not sure the tests even run these days (`rspec` appears to just leave everything as "pending", and `rake test` barfs).

Reply:
Ah ok, we can always add it back if necessary.